gnd/src/main/java/com/google/android/gnd/model/form/Form.java

```diff
@@ -35,6 +37,12 @@
 
   public abstract ImmutableList<Element> getElements();
 
+  public ImmutableList<Element> getElementsSorted() {
```

Comment:
How about `getElementsSorted()` --> `getSortedElements()`?

Reply:
The naming was chosen to emphasize that this is a variant of the `getElements()` method -  it also returns elements, but sorted. I've seen this naming convention used elsewhere but can't find it now. I don't feel strongly about it though. Is there precedency for this pattern that you know of?

Reply:
I don't know if there is a rule/pattern for that. It just seemed much more natural to add "sorted" first. I am fine with either as it is just a minor nit. Let's keep it the same way.